requirements-analysis.txt

```diff
@@ -0,0 +1 @@
+prospector[with_dodgy,with_frosted,with_mccabe,with_pep257,with_pep8,with_pylint,with_pyroma,with_vulture]
```

Comment:
what is this file

Reply:
Ah I was just trying something. Not required. will remove this.